src/Report/ReportSummary.php

```diff
@@ -40,21 +40,37 @@
     private $isDryRun = false;
 
     /**
-     * @var float
+     * @var int
      */
     private $memory;
 
     /**
-     * @var float
+     * @var int
      */
     private $time;
 
     /**
-     * @return ReportSummary
-     */
-    public static function create()
-    {
-        return new self();
+     * @param array $changed
+     * @param int   $time
+     * @param int   $memory
+     * @param bool  $addAppliedFixers
+     * @param bool  $isDryRun
+     * @param bool  $isDecoratedOutput
+     */
```

Comment:
these could you a bit of explaining, like is time processing time, UNIX timestamp, is it in seconds or minisecs., same for memory; is it in bytes or bits?